pkg/collect/host_kube_ssl_cert_collector.go

```diff
@@ -0,0 +1,146 @@
+package collect
+
+import (
+	"bytes"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+)
+
+type CollectKubeSSLCertInfo struct {
+	hostCollector *troubleshootv1beta2.KubeSSLCertCollect
+	BundlePath    string
+}
+
+func (c *CollectKubeSSLCertInfo) Title() string {
+	return getCollectorName(c)
+}
+
+func (c *CollectKubeSSLCertInfo) IsExcluded() (bool, error) {
+	return isExcluded(c.hostCollector.Exclude)
+}
+
+// SSL Certificate Struct
+type KubeSSLCertCollect struct {
+	CertName         string    `json:"Certificate Name"`
```

Comment:
The project follows a `camelCase` naming convention meaning this would be `certificateName`

Reply:
ack